R/qenv-get_warnings.R

```diff
@@ -33,7 +33,9 @@ setGeneric("get_warnings", function(object) {
 })
 
 setMethod("get_warnings", signature = c("qenv"), function(object) {
-  if (all(object@warnings == "")) {
+  warnings <- lapply(object@code, "attr", "warning")
```

Comment:
Too many as.list/unlist. Please optimise
```suggestion
  warnings <- lapply(object@code, "attr", "warning")
  idx_warn <- which(sapply(warnings, Negate(is.null)))
```

Reply:
Thanks, changed in here https://github.com/insightsengineering/teal.code/pull/216/commits/7bd1c2d376e76427dab232585c06938d8e266095